app/graphql/types/account.rb

```diff
@@ -15,6 +15,63 @@ class Types::Account < Types::BaseObject
     null: true,
     description: 'The PRO subscription for this account'
 
+  #
+  # field :pro_started_at, GraphQL::Types::ISO8601DateTime,
+  #   null: true,
+  #   description: 'When your pro subscription started'
+  #
+  # # unsure if this is the right place
+  # field :pro_expires_at, GraphQL::Types::ISO8601DateTime,
+  #   null: true,
+  #   description: 'When your pro subscription expires'
+
+  field :max_pro_streak, Integer,
+    null: true,
+    description: ''
+
+  field :facebook_id, String,
+    null: true,
+    description: 'Facebook account linked to the user.'
+
+  field :twitter_id, String,
+    null: true,
+    description: 'Twitter account linked to user.'
+
+  field :sfw_filter, Boolean,
+    null: true,
+    description: 'Whether Not-Safe-for-Work content is shown.'
+
+  field :mal_username, String,
+    null: true,
+    description: 'Your MyAnimeList username.'
+
+  # field :past_names, [String, null: true],
+  #   description: 'A list of this users past names'
+
+  field :theme, Integer,
+    null: false,
+    description: 'Options are the Devil! - Josh'
+
+  field :language, String,
+    null: true,
+    description: 'Primary language for this user.'
+
+  field :title_language_preference, String,
```

Comment:
Is it possible to have open-ended enums? I.e "canonical", "romanized", "english" documented but "french" would also be valid for the GraphQL schema?

Reply:
Is it even worth making an Enum? This is just a query...

Reply:
Yes, enums are worth it even when they're only used once.  They tell API users what is possible in a given field.

How about we even match this better to what the TitlesList field looks like?

```graphql
enum TitleLanguagePreference {
  CANONICAL
  LOCALIZED_EN
  LOCALIZED_EN_JP
}
```

Or we could do

```graphql
enum TitleLanguagePreference {
  CANONICAL
  ROMANIZED
  LOCALIZED
}
```

And in `LOCALIZED` mode actually prefer the User's interface language!

I'd love to get your thoughts @wopian 🤔

Reply:
The canon, Roman and local enum would be good.